server/service/src/item/mod.rs

```diff
@@ -75,6 +82,31 @@ pub trait ItemServiceTrait: Sync + Send {
     ) -> Result<String, DeletePackagingVariantError> {
         delete_packaging_variant(ctx, input)
     }
+
+    fn get_bundled_items(
```

Comment:
There's no query for this at the graphql layer, I figured we might want to do a loader of item_variant like we did for item_variant off item?

I figure we can leave that till we do the UI part...